### DISCUSSION TITLE
Is there a "scroll bar" feature for the mark_text encoding?

### DISCUSSION THREAD

**Discussion by lenard510** [vega/altair#3194] [General]
I am looking for a way to scroll through 10,000 lines of free text, each text takes a line in the dataframe.  The plan is to let another bar chart interact with this "text only" chart. When one bar is clicked, The text chart would populate texts about that category only.  

So far I could only set a row_count limit <50.  Without the scroll bar feature, the text is not readable and begin to squeeze into each other, if I don't set up line limit.  The goal is to let all texts populate and readable. 

Here is the screen between a limit<50 and limit <500 . 
![image](https://github.com/altair-viz/altair/assets/97851453/916b233f-a19b-4541-ae85-5c67110e92e1)
![image](https://github.com/altair-viz/altair/assets/97851453/dd7e26c0-e6cd-4237-a867-01806a54975b)

Here is a sample reference of the code I use.  
  # create text chart 
    click = alt.selection_multi(encodings=['color'])    #interactive filter element
    df_text = alt.Chart(df_COMMENT
                        ,title=alt.Title(
                               'Comments', anchor='middle' )
     ).mark_text(  align="left",
                          baseline="middle",
                          width=1000,
                          height=500,
                          fontSize=12
      ).encode(  y=alt.Y('row_number:O').axis(None)   ).transform_filter( click )  .transform_window(
    row_number='row_number()'
).transform_filter(
    alt.datum.row_number < 500
).properties(
        width=1000, height=alt.Step(10)
    )

    # read the text field
    aspect = df_text.encode(text=alt.Text('ASPECT')).properties(
                width=300, height=500
            ).interactive()

Any suggestions are appreciated , thank you.

**Comment by binste** [comment]
[This SO response](https://stackoverflow.com/a/68826726) shows two ways to do it, although the code is in Vega-Lite but maybe you can get some inspiration from there. If you can control the frontend code where your chart is displayed, I'd recommend the second option with  setting a height and then setting `overflow: scroll`. I don't think there is a pure Altair/Vega-Lite way of adding a scroll bar.

**Reply by lenard510** [reply]
Thank you @binste . I've been chasing down the response link and frontend code part. It seems like a rabbit hole to my level of coding exp. I would have to look at other methods at this point. It will be great to have this nice feature available in the future release of the altair package.
